### PR TITLE
Backend signup 2

### PR DESCRIPTION
**Pull request by martinlabate**
### Summary
Implemented sign up backend

### Cards
* https://trello.com/c/8PpytXcJ/16-n-12-backend-signup

### CODE REVIEW

File: app/src/main/java/com/github/martinlabate/ParseService.java [hunk 1]
```diff
@@ -12,4 +17,15 @@
 
     @GET("login")
     Call<UserInformation> logIn(@Query("username") String username, @Query("password") String password);
+
+
+    @Headers({
```
Comment (luksdlt92) [inline app/src/main/java/com/github/martinlabate/ParseService.java]: ?

**Comment by luksdlt92** [comment]
@martinlabate Check the corrections in Login backend and apply them here.

**Comment by emalamela** [comment]
It looks nice! Way to go!

giphy

Just squash it and i'll aprove it!

**Comment by emalamela** [comment]
:+1:

**Pull request merged**